src/main/java/Translator.java

```diff
@@ -105,7 +119,7 @@ protected void processLine(String line) {
         Scale scale = new Scale();
         for (int k = 0; k < itemCount; k++) {
             String label = row[k];
-            int value = Integer.parseInt(row[k + itemCount]);
+            CellContent value = new CellContent(row[k + itemCount]);
```

Comment:
yes, that's correct. Maybe I should have asked the following question: Should we introduce some kind of exception handling in case someone defines an invalid translation file (i.e. has not 2*n entries per row)?